lib/python/bind_data_array.h

```diff
@@ -243,4 +243,36 @@ void bind_data_array_properties(py::class_<T, Ignored...> &c) {
   c.def_property_readonly(
       "masks", [](T &self) -> decltype(auto) { return self.masks(); },
       R"(Dict of masks.)");
+  c.def("drop_coords",
```

Comment:
@SimonHeybrock 
Indeed xarray was using iterable as an input. I could do something like this.